akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -231,8 +231,13 @@ Scala
 Java
 :  @@snip [BasicPersistentBehaviorsTest.java]($akka$/akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorsTest.java) { #tagging }
 
-## Current limitations
-
-* The `PersistentBehavior` can't be wrapped in other behaviors, such as `Behaviors.setup`. See [#23694](https://github.com/akka/akka/issues/23694)
+## Wrapping Persistent Behaviors
+When creating/setting up `PersistentBehavior`, it is possible to wrap `PersistentBehaviors` in
```

Comment:
wrap a `PersistentBehavior` rather than PersistentBehaviors as it is just the factory.